src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.url = url + slug;
+  }
+
+  get commentButton() {
+    return '[Publish comment] button';
+  };
+
+  clickOnCommentButton() {
+    return `Click on the ${this.commentButton}`;
```

Comment:
Similar to the `clickOnLogo` method, the `clickOnCommentButton` method should likely interact with the DOM or call other methods to simulate clicking on a button. If this is a placeholder, consider adding a comment to indicate this.